src/doc/trpl/hello-cargo.md

```diff
@@ -33,7 +33,8 @@ $ mv main.rs src/main.rs
 ```
 
 Note that since we're creating an executable, we used `main.rs`. If we
-want to make a library instead, we should use `lib.rs`.
+want to make a library instead, we should use `lib.rs`. This convention is required
+for Cargo to successfully compile our projects. 
```

Comment:
Ah, ok. I'm new to Rust so sorry for the misunderstanding. I've added that and committed the change.